pkg/plugins/generic/generic_plugin.go

```diff
@@ -20,7 +20,10 @@ type GenericPlugin struct {
 	LoadVfioDriver uint
 }
 
-const scriptsPath = "bindata/scripts/enable-kargs.sh"
+const (
+	scriptsPath          = "bindata/scripts/enable-kargs.sh"
+	commandNotFoundError = 127
```

Comment:
Where this const is used?

Reply:
Nowhere anymore. Was using it for checking the error code but realized the daemon image is being built with go 1.10 which does not yet have `ExitCode()` method on  `ExitError`s.
Will remove it.